_posts/2020-04-16-remote-junior-developer-tips.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: 'Tips From One Remote Junior Developer to Another.'
+date: 2020-04-19 10:30:00
+categories: ["junior", "remote"]
+author: fionaDL
+---
+
+Becoming the Apprentice, and then Junior Developer at OmbuLabs was both exciting and nerve racking. Learning a new company can be challenging, especially when the skills are new and the work is remote.
+
+Fortunately OmbuLabs made the journey a smooth and easy process. Here are some of the things that have been working.
+
```

Comment:
@FionaDL You're missing the `<!--more-->` tag here. It is necessary so it doesn't show the entire article in the index page.

```suggestion
<!--more-->
```